components/definition/cellRules/sections/NetPyNESection.js

```diff
@@ -123,9 +137,12 @@ export default class NetPyNESection extends React.Component {
     }
     else if (this.state.sectionId == "Topology") {
       content = (<div>
-        <NetPyNEField id="netParams.cellParams.secs.topol.parentSec" >
-          <PythonControlledTextField
+        <NetPyNEField id={"netParams.cellParams.secs.topol.parentSec"} >
+          <PythonMethodControlledSelectField
             model={"netParams.cellParams['" + this.props.cellRule + "']['secs']['" + this.props.name + "']['topol']['parentSec']"}
+            method={"netpyne_geppetto.getAvailableSections"}
+            postProcessItems={this.postProcessMenuItems}
+            multiple={false}
```

Comment:
This is the value by default. No need to add it, right?

Reply:
ok

Reply:
no problem with the section assignment... as long as the CellRule has a section named "Section 2", the assignment will be correct.